moveit_simple/src/moveit_simple.cpp

```diff
@@ -895,7 +909,30 @@ bool Robot::toJointTrajectory(const std::string traj_name,
   return true;
 }
 
+bool Robot::computeIKTransforms()
+{
+  if (!virtual_robot_state_->knowsFrameTransform(ik_base_frame_))
+  {
+    ROS_ERROR_STREAM("Unable to find the transformation to frame: "
+      << ik_base_frame_ << " in group: " << joint_group_->getName()
+      << " IK solutions will not be found");
+    return false;
+  }
+
+  if (!virtual_robot_state_->knowsFrameTransform(ik_tip_frame_))
+  {
+    ROS_ERROR_STREAM("Unable to find the transformation to frame: "
+      << ik_tip_frame_ << " in group: " << joint_group_->getName()
+      << " IK solutions will not be found!");
+    return false;
+  }
 
+  std::vector<std::string> link_names = joint_group_->getLinkModelNames();
+  tool0_to_tcp_ = virtual_robot_state_->getFrameTransform(link_names.back()).inverse()
```

Comment:
Does this require the frame transform from `tool0_to_tcp_` to be fixed?

I would use another name other than `tool0`.  The name is inconsistent with the rest of the class.